test/test_transforms_v2_refactored.py

```diff
@@ -68,7 +70,8 @@ def _script(fn):
     try:
         return torch.jit.script(fn)
     except Exception as error:
-        raise AssertionError(f"Trying to `torch.jit.script` '{fn.__name__}' raised the error above.") from error
+        name = getattr(fn, "__name__", fn.__class__.__name__)
```

Comment:
Good catch! Should we rename the parameter to `obj` or the like, since the input is no longer just a function?